shared/contracts/BaseTemplate.sol

```diff
@@ -65,20 +65,39 @@ contract BaseTemplate is APMNamehash, IsContract {
         acl.createPermission(address(this), app, permission, address(this));
     }
 
+    function _createPermissions(ACL acl, address[] grantees, address app, bytes32 permission, address manager) internal {
+        acl.createPermission(grantees[0], app, permission, address(this));
+        for (uint256 i = 1; i < grantees.length; i++) {
+            acl.grantPermission(grantees[i], app, permission);
+        }
+        acl.revokePermission(address(this), app, permission);
```

Comment:
It seems slightly confusing to me that this function works different than the previous one here (template's permission removed).

Reply:
not using the remove permission helper here since we don't want to do that actually, we just want to remove the permission for the template, but to keep the one granted to the grantees, thus we are revoking the template's permission and then changing the manager

Reply:
Yes, but I'm not talking about the helper (sorry my explanation was awful, haha). I mean that after calling `_createPermission` the template will hold the permission, while after calling `_createPermissions` (with a very similar name) it won't. That's what I find confusing.

Reply:
I see that names were changed in #102 , now it's less confusing.